src/context/AppProvider/AppContext.tsx

```diff
@@ -213,6 +214,13 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
           case dogeChainId:
           case bchChainId:
           case cosmosChainId:
+          case bscChainId:
```

Comment:
fallthrough?

Reply:
The `ethChainId` has slightly more logic - if we're fine with turning `no-duplicate-case` off here, might as well!

Reply:
was just thinking with the avalanche chain id

Reply:
Absolutely - 7a101c1691